### ISSUE TITLE
Unable to open .bcup file from Google Drive

### ISSUE THREAD

**Issue by tigrouind** [buttercup/buttercup-desktop#1388]
I have a password file (first.bcup) shared between multiple computers using google drive, it works fine.
On a one computer, I have created another password file (second.bcup), which is also on google drive. It works OK.
On another computer, I want to use second.bcup. However in the panel that open after authentication, only first.bcup is suggested.

There is a workaround suggested by some users here which is to download bcup file, export to csv, create new empty bcup on google drive (instead of add) and import the csv. If I do that I end up with two second.bcup files, which is not what I want.

**Comment by tigrouind** [comment]
Some additional tests : 
- I have created a new folder "Test" and a new "Test.bcup" file inside, using the mobile application (AFAIK creating folders is not possible on Windows desktop). On another computer, the folder "Test" will now appear along with the "Test.bcup" file (there is other folders on the Google Drive which are not visible). So it looks like there are files / folders that have special permissions that allow Buttercup to view them.
- Using Google Drive, I have placed my password2.bcup file into "Test" folder. Still, it will not be visible.

**Comment by Arlet02** [comment]
I have the same problem and I haven't been able to solve it.

**Comment by Arlet02** [comment]
Suddenly, Buttercup stopped opening my vault that was on Google Drive and gave me the error that I left here, now when I try to connect it again it is not possible because the file does not appear in Google Drive
![Captura de pantalla 2024-11-14 205618](https://github.com/user-attachments/assets/c0327fa6-2966-4a19-ae22-7265328c143b)

**Comment by SLZ7** [comment]
I think I might be having the same problem
![image](https://github.com/user-attachments/assets/0ae99c3e-2090-4935-a49d-0572a3cf0593)

**Comment by vkcorp** [comment]
I also had same problem. I guess there maybe something changed on Google..
After removing my vault and add it as new (granting permission from Google), then it's resolved !
Sigh.. supprised.

**Comment by ccarranza78** [comment]
I had the same problem. The workaround is adding the vault again to buttercup.

**Comment by tehsphinx** [comment]
Yeah... same problem here. And I haven't found a way to re-add a vault on iPhone. Desktop I can use Google Drive Sync and then open the local file in buttercup, but on iPhone that is not possible. At least I haven't found a way to do it.

**Comment by tehsphinx** [comment]
The recently added restrictions for Google Drive to only show files created by buttercup seriously messed things up (IMO).
